### PR TITLE
add informers to new backup controllers (gate by restart)

### PR DESCRIPTION
**Pull request by tjungblu**

### CODE REVIEW

File: pkg/operator/ceohelpers/already_synced_informer.go [hunk 1]
```diff
@@ -0,0 +1,21 @@
+package ceohelpers
+
+import (
+	"k8s.io/client-go/tools/cache"
+)
+
+// AlreadySyncedInformer wraps an ordinary cache.SharedInformer, but allows us to fake HasSynced results.
+// This is to allow controllers to proceed when an informer could not sync (eg when the API isn't available).
+type AlreadySyncedInformer struct {
```
Comment (tjungblu) [inline pkg/operator/ceohelpers/already_synced_informer.go:9]: leaving this in here for now, might need to reuse this for the optional machine API

File: pkg/operator/starter.go [hunk 2]
```diff
@@ -510,28 +527,6 @@ func RunOperator(ctx context.Context, controllerContext *controllercmd.Controlle
 	return nil
 }
 
-// GetReleaseVersion gets the release version string from the env
-func GetReleaseVersion(c v1.ClusterOperatorInterface) (string, error) {
```
Comment (hasbro17) [inline pkg/operator/starter.go:514]: Good catch, I guess this wasn't being used anymore.

**Comment by tjungblu** [comment]
/override ci/prow/e2e-aws-ovn-single-node

unrelated failure from machine API with:
> {  fail [github.com/openshift/origin/test/extended/single_node/topology.go:97]: machine-os-builder in openshift-machine-config-operator namespace expected to have 1 replica but got 0

**Pull request merged**